Sources/PortalSwift/Core/Mpc/Mobile/EnclaveMobileWrapper.swift

```diff
@@ -41,7 +43,12 @@ class EnclaveMobileWrapper: MPCMobile {
       return encodeErrorResult(id: "INVALID_PARAMETERS", message: "Invalid parameters provided")
     }
 
-    guard let url = URL(string: "https://\(enclaveMPCHost)/v1/sign") else {
+    var path = "/v1/sign"
+    if isRaw ?? false, let curve {
```

Comment:
If `isRaw` is true but curve wasn't supplied, we should probably throw an exception instead of defaulting to `/v1/sign`